src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    public void draw();
```

Comment:
its not neccesary to explicitly declare access modifier, its always public for interface methods
```suggestion
    void draw();
```